fortanix-vme/fortanix-vme-abi/src/lib.rs

```diff
@@ -54,6 +1250,658 @@ pub enum Addr {
     },
 }
 
+/// Serializes an `Addr` value. We can't rely on the `serde` `Serialize` macro as we wish to use
```

Comment:
Nit: I'd write "We can't rely on the `serde_derive` macro ..." since technically `Serialize` is not a macro.